src/box/lua/load_cfg.lua

```diff
@@ -309,6 +311,20 @@ local function check_replicaset_uuid()
     end
 end
 
+local function cfg_set_wal_ext()
+    if box.cfg.wal_ext ~= nil then
+        if box.internal.space.wal_ext == nil then
+            box.error(box.error.UNSUPPORTED, "Community edition",
+                      "WAL extensions")
+        end
+        local status, err = pcall(box.internal.space.wal_ext.configure,
```

Comment:
There's no point in using the `space.wal_ext` namespace, because there's no nothing more there to be. Let's rename the function to `box.internal.cfg_set_wal_ext` to be consistent with other configuration functions. Also, the proper error'd better be set right from that function, and the `wal_ext == nil` case should be handled there as well - it'll make it easier to migrate this code to CE in future.

```diff
diff --git a/src/box/lua/load_cfg.lua b/src/box/lua/load_cfg.lua
index 11cc52aff50d..5584e017d439 100644
--- a/src/box/lua/load_cfg.lua
+++ b/src/box/lua/load_cfg.lua
@@ -35,6 +35,16 @@ local ifdef_feedback_set_params =
     private.feedback_daemon ~= nil and
     private.feedback_daemon.set_feedback_params or nil
 
+-- Stub implementation of the WAL extension configuration function.
+if private.cfg_set_wal_ext == nil then
+    private.cfg_set_wal_ext = function()
+        if box.cfg.wal_ext ~= nil then
+            box.error(box.error.UNSUPPORTED, "Community edition",
+                      "WAL extensions")
+        end
+    end
+end
+
 -- all available options
 local default_cfg = {
     listen              = nil,
@@ -366,6 +376,7 @@ local dynamic_cfg = {
     sql_cache_size          = private.cfg_set_sql_cache_size,
     txn_timeout             = private.cfg_set_txn_timeout,
     txn_isolation           = private.cfg_set_txn_isolation,
+    wal_ext                 = private.cfg_set_wal_ext,
 }
 
 -- dynamically settable options, which should be reverted in case
```

Reply:
Applied this diff. Also according to our f2f discussion I've added one more patch which moves cfg_get() to global namespace.